### DISCUSSION TITLE
New Want: I want elements with the CSS "resize" property to have a ::handle pseudo element

### DISCUSSION THREAD

**Discussion by Kilian** [WebWeWant/webwewant.fyi#716] [Wants]
My proposal: 

---
title: I want elements with the CSS "resize" property to have a ::handle pseudo element
date: 2021-08-02
submitter: Kilian Valkhof
---

With the CSS property `resize` set to `horizontal` or `vertical` on elements with overflow:hidden, you get a little resize control in the bottom right of the element with which to resize it.

I think this should be a pseudo element just like `::marker` for list items and summary elements. This would allow it to be styled in ways that make sense for the current element as well as the current direction of resize when it's limited to a single dimension.

`::handle` as a psuedo-element to achieve this makes sense to me. I ran into this when writing [this article](https://polypane.app/blog/faking-container-queries-with-css-grid/), where I would really prefer to have a differently styled handle for the resizable examples.

**Comment by gregwhitworth** [comment]
Thanks for filing this, there is also an open issue for a similar request on Open UI [here](https://github.com/WICG/open-ui/issues/255). We'd be more than happy to have your help with the necessary research to lead the way for possible standardization.
